packages/flex-plugin-e2e-tests/src/tests/step002.ts

```diff
@@ -1,65 +1,66 @@
 /* eslint-disable import/no-unused-modules, sonarjs/no-duplicate-string */
-import { TestSuite, TestParams } from '..';
+import { TestSuite, TestParams } from '../core';
 import { spawn, logResult, assertion, joinPath, writeFileSync } from '../utils';
 
 // Create a plugin
-const testSuite: TestSuite = async (params: TestParams): Promise<void> => {
-  const twilioCliResult = await spawn('twilio', ['flex:plugins:create', params.plugin.name]);
+const testSuite: TestSuite = async ({ scenario, config }: TestParams): Promise<void> => {
+  const twilioCliResult = await spawn('twilio', ['flex:plugins:create', scenario.plugin.name]);
   logResult(twilioCliResult);
 
   // Assert files/directories exist
-  assertion.fileExists([params.plugin.dir], 'Plugin directory does not exist');
-  assertion.fileExists([params.plugin.dir, 'src']);
-  assertion.fileExists([params.plugin.dir, 'src', 'components']);
-  assertion.fileExists([params.plugin.dir, 'src', 'components', '__tests__']);
-  assertion.not.dirIsEmpty([params.plugin.dir, 'src', 'components', '__tests__']);
-  assertion.fileExists([params.plugin.dir, 'public']);
-  assertion.fileExists([params.plugin.dir, 'package.json']);
-  assertion.fileExists([params.plugin.dir, 'webpack.config.js']);
-  assertion.fileExists([params.plugin.dir, 'webpack.dev.js']);
-  assertion.fileExists([params.plugin.dir, 'jest.config.js']);
-  assertion.fileExists([params.plugin.dir, 'public', 'appConfig.js']);
-  assertion.fileExists([params.plugin.dir, 'public', 'appConfig.example.js']);
+  assertion.fileExists([scenario.plugin.dir], 'Plugin directory does not exist');
+  assertion.fileExists([scenario.plugin.dir, 'src']);
+  assertion.fileExists([scenario.plugin.dir, 'src', 'components']);
+  assertion.fileExists([scenario.plugin.dir, 'src', 'components', '__tests__']);
+  assertion.not.dirIsEmpty([scenario.plugin.dir, 'src', 'components', '__tests__']);
+  assertion.fileExists([scenario.plugin.dir, 'public']);
+  assertion.fileExists([scenario.plugin.dir, 'package.json']);
+  assertion.fileExists([scenario.plugin.dir, 'webpack.config.js']);
+  assertion.fileExists([scenario.plugin.dir, 'webpack.dev.js']);
+  assertion.fileExists([scenario.plugin.dir, 'jest.config.js']);
+  assertion.fileExists([scenario.plugin.dir, 'public', 'appConfig.js']);
+  assertion.fileExists([scenario.plugin.dir, 'public', 'appConfig.example.js']);
```

Comment:
Why not just destruct scenario further?

Reply:
To make it clear that the `plugin` is part of the scenario we are running, vs a local variable.